be/src/storage/persistent_index.cpp

```diff
@@ -706,7 +719,17 @@ Status PersistentIndex::load(const PersistentIndexMetaPB& index_meta) {
     });
     std::string l0_index_file_name = _get_l0_index_file_name(_path, start_version);
     RETURN_IF_ERROR(block_mgr->open_block(l0_index_file_name, &rblock));
-    // TODO: load snapshot first
+    // Assuming that the snapshot is alaways at the beginning of index file,
```

Comment:
spelling error of always.

Reply:
done, change in https://github.com/StarRocks/starrocks/pull/3654/commits/338e49b76ac9e44b64bf933dba1e5c0d0646e198